react/components/ComponentEditor/index.tsx

```diff
@@ -273,7 +262,10 @@ class ComponentEditor extends Component<
             deepProperties,
           )),
         ...(itemsProperties &&
-          map(item => getDeepUiSchema(item), itemsProperties)),
+          map(
+            item => getDeepUiSchema(item),
+            itemsProperties)
+          ),
```

Comment:
:pick: I think the parentheses are not formatted correctly, perhaps the following would be better:

```js
...(itemsProperties &&
  map(
    item => getDeepUiSchema(item),
    itemsProperties
  )
),
```